src/supermarket/db/seeds.rb

```diff
@@ -88,7 +88,7 @@
         description: Faker::Lorem.sentences(1).first,
         license: 'MIT',
         tarball: File.open('spec/support/cookbook_fixtures/redis-test-v1.tgz'),
-        readme: File.read('../../README.md'),
+        readme: '# This is a README',
```

Comment:
Removing another file reference that is reaching outside of the supermarket app root tree. For a long time the project's README has been used as a fixture README for development environment seeding of test cookbooks. This stops that. If we opt to have more complicated test READMEs, they should probably live in the test fixtures directory.